safe/definitions/default_settings.py

```diff
@@ -23,7 +23,7 @@
     'ISO19115_LICENSE': 'Free use with accreditation',
 
     'keywordCachePath': abspath(join(
-        expanduser('~'), '.inasafe', 'keywords.db')),
+        expanduser('~'), 'inasafe', 'metadata.db')),
```

Comment:
Ah yes, thanks. I copy and paste from the metadata. But it seems I pasted the wrong one.